lib/internal/process/pre_execution.js

```diff
@@ -530,6 +530,7 @@ function initializePermission() {
     const warnFlags = [
       '--allow-addons',
       '--allow-child-process',
+      '--allow-net-dns',
```

Comment:
I don't think we should emit a warning when `--allow-net-dns` is used. It won't open breaches to bypass permission model restrictions as the other CLI args.